### PR TITLE
Murphant knowledge3

### PR DESCRIPTION
**Pull request by LeMurphant**
TODOs that are left that I can't deal with
        {/* TODO: add video png*/}
          // TODO: what's the link for agi safety core? I never heard of them
        // TODO: fix additionalInfo appearing for each link
      {/* TODO: Boost your learning efforts, Nemo's job */}

### CODE REVIEW

**Review by mruwnik — APPROVED** [review]

**Comment by bryceerobertson** [comment]
AGI Safety Core: https://x.com/i/lists/1185207859728076800

**Comment by LeMurphant** [comment]
PR seems to fail due to issues in stampy-redesign

File: app/routes/how-can-i-help.knowledge.tsx [hunk 1]
```diff
@@ -109,7 +202,15 @@ const DiveDeeper = () => (
...
-        // TODO: add "full list of courses"
+        additionalInfo={
+          <>
+            Or, browse our{' '}
+            <a href="https://www.aisafety.com/courses" className="small-bold teal-500">
+              full list of courses
+            </a>
+          </>
+        }
+        // TODO: fix additionalInfo appearing for each link
```
Comment (melissasamworth) [inline app/routes/how-can-i-help.knowledge.tsx:213]: I'm not sure what you mean by this, maybe merge and then I'll see

**Review by melissasamworth — APPROVED** [review]

File: app/routes/how-can-i-help.knowledge.tsx [hunk 2]
```diff
@@ -79,10 +85,97 @@ const NewToAISafety = () => (
...
-    <DropDown title="I want something I can subscribe to, so I can learn more about AI safety consistently">
-      <p>TODO fill this</p>
+
+    <DropDown title="I want something I can subscribe to (Youtube channels, newsletters, etc.)">
+      <HelpItem
+        title="Youtube"
+        links={[
+          {
+            tag: 'Top recommendation',
+            title: 'Rob Miles AI Safety',
```
Comment (bryceerobertson) [inline app/routes/how-can-i-help.knowledge.tsx:117]: "Robert"

File: app/routes/how-can-i-help.knowledge.tsx [hunk 3]
```diff
@@ -79,10 +85,97 @@ const NewToAISafety = () => (
...
+            action: 'https://www.youtube.com/c/robertmilesai',
+          },
+        ]}
+      >
+        We recommend the Youtube channel Robert Miles AI Safety, which presents complex AI safety
```
Comment (bryceerobertson) [inline app/routes/how-can-i-help.knowledge.tsx:122]: "YouTube"

**Review by bryceerobertson — APPROVED** [review]

File: app/routes/how-can-i-help.knowledge.tsx (hunk 1 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.knowledge.tsx:213]: You can see that the "Or, browse etc." appears twice, once for each link

File: app/routes/how-can-i-help.knowledge.tsx (hunk 2 above)
Comment (LeMurphant) [reply app/routes/how-can-i-help.knowledge.tsx:117]: I don't know if Rob goes by "Robert" officially, but I changed all instances of "Rob" to "Robert"

Comment (melissasamworth) [reply app/routes/how-can-i-help.knowledge.tsx]: His yt channel is "Robert" but everywhere else I've seen/heard "Rob"

**Review by melissasamworth — APPROVED** [review]

File: app/routes/how-can-i-help.knowledge.tsx (hunk 2 above)
Comment (bryceerobertson) [reply app/routes/how-can-i-help.knowledge.tsx]: Yeah I think it's just his YouTube. His Discord server for example is Rob. I would probably refer to him as Rob everywhere except when naming his YouTube

**Pull request merged**